### ISSUE TITLE
Contents of the window doesn't render on Mac

### ISSUE THREAD

**Issue by darwingr** [ThreeFourSeven/Database-Gui#1]
Mac OS X version 10.11.6
Java version 8 update 161
Eclipse Version: Oxygen.2 Release (4.7.2)

### Console message
```
<terminated>  GameManager [Java Application]
------------------------------------------------------------------
Exception in thread "main" java.lang.ExceptionInInitializerError
	at game.gui.Button.render(Button.java:59)
	at game.gui.Tab.render(Tab.java:33)
	at game.gui.Gui.render(Gui.java:57)
	at game.GameManager.render(GameManager.java:28)
	at engine.GameContainer.run(GameContainer.java:76)
	at java.lang.Thread.run(Thread.java:745)
	at engine.GameContainer.start(GameContainer.java:28)
	at game.GameManager.main(GameManager.java:34)
Caused by: java.lang.IllegalArgumentException: input == null!
	at javax.imageio.ImageIO.read(ImageIO.java:1348)
	at engine.gfx.Image.<init>(Image.java:20)
	at engine.gfx.Font.<init>(Font.java:13)
	at engine.gfx.Font.<clinit>(Font.java:5)
	... 8 more
```

### Screenshot
![screenshot 2018-02-03 17 31 54](https://user-images.githubusercontent.com/6166398/35990272-e09d68a8-0cd9-11e8-8628-fc20610ca283.png)

### Guesses
I think Seven mentioned this. It looks like it's related to it expecting a certain system font.
Apparently java has some default built in fonts that allow the app to run on any system, 
regardless of the system fonts available.

The first reply here mentions this: [Why do the fonts in the Java application I'm running...](https://superuser.com/questions/112326/why-do-the-fonts-in-the-java-application-im-running-look-different-in-mac-os-x).

**Comment by darwingr** [comment]
Mike is also having this error, Not sure if it's run for hime before.
That would be good to know.

**Comment by darwingr** [comment]
![screenshot 2018-02-13 20 20 43](https://user-images.githubusercontent.com/6166398/36181615-ec971a3e-10fb-11e8-954c-e9d349b4cd5c.png)
IT WORKS!

It was just a file path missing in the .classpath

**Comment by darwingr** [comment]
Fixed in db68f68
